packages/playground/remote/src/lib/playground-mu-plugin/0-playground.php

```diff
@@ -176,3 +176,51 @@ function networking_disabled() {
 		return $transports;
 	});
 }
+
+// Configure error logging
+$log_file = WP_CONTENT_DIR . '/debug.log';
+error_reporting(E_ALL);
+define('ERROR_LOG_FILE', $log_file);
+ini_set('error_log', $log_file);
+
+/**
+ * Get OpenTracing severity number from PHP error code
+ *
+ * @param int $code PHP error code
+ * @return int OpenTracing severity number
+ */
+function get_severity_number($code)
+{
+	// @TODO Update numbers
+	switch ($code) {
+		case E_ERROR:
+		case E_CORE_ERROR:
+		case E_COMPILE_ERROR:
+		case E_USER_ERROR:
+			return 17;
```

Comment:
What's `17`? Let's give it a name and document it a bit more.

Reply:
Some Open Telemetry magic. I changed it for strings to make sense.